src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -66,7 +67,30 @@ class CoreWorkerDirectTaskSubmitter {
         lease_timeout_ms_(lease_timeout_ms),
         local_raylet_id_(local_raylet_id),
         actor_create_callback_(std::move(actor_create_callback)),
-        cancel_retry_timer_(std::move(cancel_timer)) {}
+        cancel_retry_timer_(std::move(cancel_timer)),
+        pipeline_fullness_(pipeline_fullness) {
+          char *pip_f_env = getenv("PIPELINE_FULLNESS_PARAM");
```

Comment:
The convention is to set parameters like these using the system config. [Here](https://github.com/ray-project/ray/blob/master/src/ray/common/ray_config_def.h#L46) is an example for defining the value, and [here](https://github.com/ray-project/ray/blob/master/python/ray/tests/test_actor_failures.py#L282) is an example for setting it when initializing Ray. A default value of 1 makes sense, so that it doesn't modify existing behavior.